app/server/lib/OIDCConfig.ts

```diff
@@ -173,6 +178,9 @@ export class OIDCConfig {
     this._protectionManager = new ProtectionsManager(enabledProtections);
 
     this._redirectUrl = new URL(CALLBACK_URL, spHost).href;
+    custom.setHttpOptionsDefaults({
+      ...(httpTimeout !== undefined ? {timeout: httpTimeout} : {}),
```

Comment:
Why not more simply:
```
if (httpTimeout !== undefined) {
  custom.setHttpOptionsDefaults({timeout: httpTimeout});
}
```

Reply:
Hi @dsagal!

For slightly convenient reasons:
 1. I don't have to care whether the function should have been called or not in the test, it should if no exception has been raised: https://github.com/gristlabs/grist-core/pull/1177/files#diff-0467f3ff93b79307ee0beb1d72eea81866d60efa1efcfb8937d7154117e895f4R238
 2. I think of passing later more options to fix #942 

If these are bad reasons to you, I may apply the change you request, I don't have much strong opinions.

Reply:
Yes, I agree with what you tell (we miss integration tests for this part :grimacing:).
I prefer not taking any assumption (for now and the future) and keep this part as it is.